scripts/apps/authoring/authoring/directives/ItemCarouselDirective.js

```diff
@@ -46,19 +50,25 @@ export function ItemCarouselDirective($timeout) {
                 scope.carouselItems = _.sortBy(_.filter(items, (item) => item[item.fieldId]),
                     [(item) => item[item.fieldId].order]);
 
-                // On first opening, wait a little for image size to be calculated
-                let interval = 500;
-
                 if (carousel) {
-                    interval = 0;
                     carousel.trigger('destroy.owl.carousel');
                 }
 
-                if (items.length > 1) {
-                    $timeout(() => {
-                        initCarousel();
-                    }, interval, false);
+                const carouselImages = Array.from(document.querySelectorAll(`${carouselContainerSelector} img`));
```

Comment:
just wondering if this works for multiple multi media fields at the same time? shouldn't it use `elem.find` rather?

Reply:
You're right, I've pushed a fix.